lib/auth/getKerberos.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const requireOptional = require('require_optional');
```

Comment:
can't we just put this into `lib/utils` similar to `retrieveBSON`? I'm starting to see the possible explosive growth of tiny files on its way 😄